readconfig.py

```diff
@@ -72,11 +84,15 @@ def parse_config(text):
                 key = key.strip()
                 value = keyvalue_match.group(3)
                 if key in currentSection:
+                    origin_lineno_length, duplicated_lineno_length = str(config_lineno[key]).__len__(), str(lineno).__len__()
+                    current_key_length = key.__len__()
```

Comment:
We need to deal with non-ASCII characters too.
![image](https://user-images.githubusercontent.com/16814706/163306324-58998f24-ae45-4746-85fb-dc56ec004d64.png)

Reply:
I think it might be hard because fonts are vary in settings. Is there any idea to measure width of text?

```python
# It's hard to measure by using utf-8 bytes.
len('A'.encode('utf-8')) # 1
len('؄'.encode('utf-8')) # 2
len('하'.encode('utf-8')) # 3
len('𐰊'.encode('utf-8')) # 4
```

Instead, How about change form of error message to below?
<img width="368" alt="image" src="https://user-images.githubusercontent.com/36349353/163330102-91b83907-4d6d-4484-a787-22231d1d62ca.png">
<img width="326" alt="image" src="https://user-images.githubusercontent.com/36349353/163330132-a9968bf1-5df0-4553-a043-0f02e721f6f0.png">